WcaOnRails/config/routes.rb

```diff
@@ -121,7 +121,6 @@
   get 'documents' => 'static_pages#documents'
   get 'delegates' => 'static_pages#delegates'
   get 'disclaimer' => 'static_pages#disclaimer'
-  get 'organizations' => 'static_pages#organizations'
```

Comment:
I don't think it's a good idea for us to change our url layout as part of this change. Google has indexed `/organizations`, and we shouldn't change that without a very compelling reason.

Reply:
That sounds good, I'll change this back!